src/webui/webui.qrc

```diff
@@ -45,5 +45,55 @@
         <file>www/public/css/style.css</file>
         <file>www/public/login.html</file>
         <file>www/public/scripts/lib/mootools-1.2-core-yc.js</file>
+        <file>www/lang/www_ar.qm</file>
```

Comment:
I would put the translations in www/locale folder.
And "www" prefix isn't really good, why not just "webui"?

Reply:
Ageed with "www -> webui".
But why "locale" is better name? It's not so obvious for me.

Reply:
But here are the translation files for some languages. So it turns out, the most correct name is "translations". 
But it's just an internal folder, so I don't really care what it's called. If you insist on "locale", then I will use it without further objection.